### PR TITLE
8 hf20 pull nordbergian changes

### LINKED ISSUE #8: Pull nordbergian changes

### PR DESCRIPTION
**Pull request by relativityboy**
Closes #8

### CODE REVIEW

**Review by roadscape — CHANGES_REQUESTED** [review]
Is this based off `master`, i.e. https://github.com/jnordberg/dsteem/compare/master...steemit:8-hf20-pull-nordbergian-changes?

 - It looks like we may still have an outdated `dist/` on this branch
 - I left a note about a dupe `create_claimed_account`
 - tests marked `disabled for steem#2936` should be safe to re-enable

Everything else looks good :+1:

File: src/steem/operation.ts [hunk 1]
```diff
@@ -57,6 +57,7 @@ export type OperationName = // <id>
     | 'comment' // 1
     | 'comment_options' // 19
     | 'convert' // 8
+    | 'create_claimed_account' // 23
```
[the remark this answers is not in the document]
Comment (relativityboy) [reply src/steem/operation.ts]: Love that you marked as dupe the incorrectly alphabetized one.

**Comment by relativityboy** [comment]
@roadscape - the not updated dist is about keeping history and review somewhat clean. We already have a `0.10.1` on npm. 

IMO, dist & docs are best updated in automation as a part of a release build.

**Review by roadscape — APPROVED** [review]
LGTM, btw tests marked `disabled for steem#2936` should be safe to re-enable

**Pull request merged**